### ISSUE TITLE
page layout messed up after upgrading to Evoq 8.5 and editing a module

### ISSUE THREAD

**Issue by StevenFouts** [dnnwerk/dnnScript#2]
We upgraded from Evoq 7.3.4 to 8.5, after applying the latest dnnScripts (TurboScripts Version 0.9.9 Rev z3) and editing a modules settings on any page, the layout is messed up. I have tracked this down to a missing comma in the script at line 12456 (after [ModuleVersion]) causing the stored procedure GetTabVersionDetailsHistory to return the ModuleVersion aliased as PaneName

WHEN I CHANGE:
```
IF {databaseOwner}[{objectQualifier}sys_currentDNNVersion]()>= 70400 Exec(N'
CREATE PROCEDURE {databaseOwner}[{objectQualifier}GetTabVersionDetailsHistory]
    @TabID   INT,
    @Version INT
AS
BEGIN
    SELECT [TabVersionDetailId],
           [TabVersionId],
           [ModuleId],
           [ModuleVersion]
           [PaneName],
           [ModuleOrder],
           [Action],
           [CreatedByUserID],
           [CreatedOnDate],
           [LastModifiedByUserID],
           [LastModifiedOnDate]
    FROM {databaseOwner}[{objectQualifier}TabVersionDetails]
    WHERE [TabVersionId] IN (SELECT [TabVersionId] FROM {databaseOwner}[{objectQualifier}TabVersions]
                              WHERE [Version] <= @Version AND [TabId] = @TabID)
    ORDER BY [CreatedOnDate]
END -- PROCEDURE
')
GO
```

TO:
```
IF {databaseOwner}[{objectQualifier}sys_currentDNNVersion]()>= 70400 Exec(N'
CREATE PROCEDURE {databaseOwner}[{objectQualifier}GetTabVersionDetailsHistory]
    @TabID   INT,
    @Version INT
AS
BEGIN
    SELECT [TabVersionDetailId],
           [TabVersionId],
           [ModuleId],
           [ModuleVersion],
           [PaneName],
           [ModuleOrder],
           [Action],
           [CreatedByUserID],
           [CreatedOnDate],
           [LastModifiedByUserID],
           [LastModifiedOnDate]
    FROM {databaseOwner}[{objectQualifier}TabVersionDetails]
    WHERE [TabVersionId] IN (SELECT [TabVersionId] FROM {databaseOwner}[{objectQualifier}TabVersions]
                              WHERE [Version] <= @Version AND [TabId] = @TabID)
    ORDER BY [CreatedOnDate]
END -- PROCEDURE
')
GO
```

Everything works as intended.

Thanks
Steven

**Comment by sleupold** [comment]
Steven,
thanks for the heads up, I'll fix the script asap and republish.
Sebastian

**Comment by sleupold** [comment]
Fixed in Release 0.9.9 Rev z4
